rmgpy/thermo/thermoengine.py

```diff
@@ -98,14 +98,15 @@ def process_thermo_data(spc, thermo0, thermo_class=NASA, solvent_name=''):
     else:
         raise Exception('thermo_class neither NASA nor Wilhoit.  Cannot process thermo data.')
 
-    if thermo.__class__ != thermo0.__class__:
-        # Compute RMS error of overall transformation
-        Tlist = np.array([300.0, 400.0, 500.0, 600.0, 800.0, 1000.0, 1500.0], np.float64)
-        err = 0.0
-        for T in Tlist:
-            err += (thermo.get_heat_capacity(T) - thermo0.get_heat_capacity(T)) ** 2
-        err = math.sqrt(err / len(Tlist)) / constants.R
-        # logging.log(logging.WARNING if err > 0.2 else 0, 'Average RMS error in heat capacity fit to {0} = {1:g}*R'.format(spc, err))
+    # if thermo.__class__ != thermo0.__class__:
```

Comment:
I know it is better to comment out so that it is easy to find in case anyone else ever needs it, but calculating the error in the fit is fairly straight forward, so if this is something we decide to implement in the future it won't take long to write from scratch, and the resulting implementation might be better for it. I think we can delete this code altogether, but if you prefer it can stay commented out

Reply:
I removed the code. You're right that it would be easy to rewrite if ever needed.